src/main/ipc.ts

```diff
@@ -126,6 +126,39 @@ ipcMain.on('open-file-dialog', async (event) => {
   }
 });
 
+ipcMain.handle('save-file', async (event, { fileData, storePath }) => {
+  try {
+    const currentDate = new Date();
+    const year = String(currentDate.getFullYear()).slice(-2);
+    const month = String(currentDate.getMonth() + 1).padStart(2, '0');
+    const day = String(currentDate.getDate()).padStart(2, '0');
+    const hours = String(currentDate.getHours()).padStart(2, '0');
+    const minutes = String(currentDate.getMinutes()).padStart(2, '0');
+    const seconds = String(currentDate.getSeconds()).padStart(2, '0');
+    const extension = 'png'; // Replace with the desired file extension
```

Comment:
is the extension always `png` in the case of pasting from clipboard?

Reply:
Thank you for pointing out. Fixed that, checking filename and determine which extension is posted:
```
const file = item.getAsFile();
const fileName = file.name; // Retrieve the filename
const fileExtension = fileName.split('.').pop(); // Extract the file extension
```

Reply:
@UdaraJay I also updated the way to handle paste using editorProps and handlePaste instead of useEffect.